src/tests/system/tests/test_sss_override.py

```diff
@@ -77,55 +77,57 @@ def test_sss_overrides__overriding_username_and_posix_attributes(client: Client,
     assert result.home == "/home/o-user1", "User's override name homedir does not match override value!"
 
 
-@pytest.mark.importance("medium")
+@pytest.mark.importance("high")
 @pytest.mark.topology(KnownTopology.LDAP)
 @pytest.mark.topology(KnownTopology.AD)
 @pytest.mark.topology(KnownTopology.Samba)
 def test_sss_overrides__overriding_group_name_and_gid(client: Client, provider: GenericProvider):
     """
     :title: Locally overriding the name and GID of a group
     :setup:
-        1. Create POSIX group "group1"
+        1. Create POSIX user "user1" as a member of group "group1"
         2. Configure SSSD with "ldap_id_mapping = false" and start SSSD
-        3. Create local override for "group1"
+        3. Create local override for "group1" and "user1"
         4. Restart SSSD, this is necessary to enable local overrides
     :steps:
         1. Lookup group name and overridden name and check gid
-        2. Override group gid to a new value, lookup group name by both names and check gid
+        2. Update group gid value for override and check group and override group
+        3. Look up members in group by overridden name
     :expectedresults:
         1. Groups are found and gid match original values
-        2. Groups are found and gid matches new overridden value
+        2. Groups are found and gid matches the new value
+        3. User overridden is a member of group
     :customerscenario: False
     """
-    provider.group("group1").add(gid=999999)
+    user = provider.user("user1").add(uid=777777, gid=777777)
+    provider.group("group1").add(gid=999999).add_member(user)
     client.sssd.domain["ldap_id_mapping"] = "False"
     client.sssd.start()
 
+    client.sss_override.user("user1").add(name="o-user1")
     group = client.sss_override.group("group1")
-
     group.add(name="o-group1")
```

Comment:
Out of curiosity: why to add an override to a user you use `client.sss_override` and to add an override to a group you use `group` itself?